src/index.ts

```diff
@@ -73,16 +76,37 @@ function createRdfEmitter(program: Program) {
             DataFactory.literal(doc)
           );
         }
+
+      },
+    });
+
+
+    // PROPERTY DEF
+    navigateProgram(program, 
+      {
+      model(m) {
+        if (m.namespace?.name === "Cadl") {
+          return;
+        }
+
+        const nameNode = nn(nameForModel(m));
         
-        // Data properties
         for (const prop of m.properties.values()) 
         {
           const propNameNode = nn(nameForProperty(prop));
-          writer.addQuad(propNameNode, nn("http://www.w3.org/1999/02/22-rdf-syntax-ns#type"), nn("owl:DataTypeProperty"));
-          writer.addQuad(propNameNode, nn("rdfs:domain"), nameNode);
 
           if (prop.type.kind === "Model") 
           {
+              if (checkIfDataProperty(prop.type)==false)

```

Comment:
```suggestion
              if (!checkIfDataProperty(prop.type))

```

Reply:
Actually you have an else block, so just swap the blocks and have `if (checkIfDataProperty(prop.type))`.